packages/modules/imodel-browser/src/containers/iModelGrid/IModelGrid.tsx

```diff
@@ -2,8 +2,26 @@
  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
-import React from "react";
+import "./IModelGrid.scss";
```

Comment:
Imo, this file shouldn't include tableView since its grid specific maybe create a separate file for tableView?? so that users who only want tableView can use that instead otherwise this fileName only implies that it provides a grid view. Maybe a common file which can then include both Grid and Table view? @raplemie what do you think?

Reply:
I was pushing the idea that this is a grid of tile, or a grid of cells... The problem is that if we have 2 completely different components, we'll have to redownload all the data when switching from table to tile, or add another layer on top to switch between the 2 and provide the "apiOverride.data" to either component, wich could be an "IModelBrowser" component, which, maybe wouldnt be a bad idea given the name of the package...

That suggests that the `IModelGrid` works as is by itself, `IModelTable` would do the same, but can be fed their complete data from the `IModelBrowser` component, and we'll need to handle the download of "more" data when we reach the bottom of either, this part is not exposed.

Reply:
@raplemie As we discused, I have moved out the logic of search bar and view buttons out of package and keep the table view inside the package and also added the lazy loading concept for the table view. I think you can review it once again and let me know if any changes and improvemnet required.